src/Components/SearchBar.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
```

Comment:
Where did this come from? It's not a part of NewRecipe.

Reply:
A SearchBar component was not a part of the work performed on this ticket. Why was this added?